testing_data/hyperparameter_opt.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Not relevant, new used file is in ivadomed/data-testing

Reply:
Delete :)?

Reply:
Done